cli/src/semgrep/commands/scan.py

```diff
@@ -469,6 +469,31 @@ def scan(
         # so the engine is properly chosen.
         run_secrets_flag = True
 
+    state = get_state()
+    state.metrics.configure(metrics)
+    state.terminal.configure(
+        verbose=verbose,
+        debug=debug,
+        quiet=quiet,
+        force_color=force_color,
+        output_format=output_format,
+    )
+
+    # Implicity enable secrets if the user has specified --allow-untrusted-validators
+    if allow_untrusted_validators and not run_secrets_flag:
+        logger.debug(
+            "Implicity enable secrets validation since allow_untrusted_validators is set."
```

Comment:
Also this should be "Implicitly enabling ..." to align with "Disabling ..." below; I also think the "Implicitly" in the log message could be dropped.

Reply:
done in [84b5eb9](https://github.com/semgrep/semgrep/pull/9382/commits/84b5eb946df55bf7daa6c413aa1cc44cda922444)